dali/base/dacsds.cpp

```diff
@@ -1887,6 +1887,19 @@ StringBuffer &CClientSDSManager::getLocks(StringBuffer &out)
     return getInfo(DIAG_CMD_LOCKINFO, out);
 }
 
+void CClientSDSManager::getLocks(CMessageBuffer &out)
+{
+    out.append((int)DAMP_SDSCMD_DIAGNOSTIC);
+    out.append((int)DIAG_CMD_LOCKINFO);
```

Comment:
I would expect this to be like the other info calls and go via getInfo().
i.e. request the serialize info, then format it.
Maybe it will become clearer when I look on.